src/Storefront/Resources/app/storefront/src/plugin/forms/form-country-state-select.plugin.js

```diff
@@ -116,19 +116,22 @@ export default class CountryStateSelectPlugin extends Plugin {
         vatIdFieldInput.removeAttribute('required');
     }
 
-    _updateZipcodeRequired(label, input, required) {
+    _updateZipcodeRequired(labels, inputs, required) {
         if (!label || !input) {
```

Comment:
Shouldn't this now check for `labels` and `inputs` instead? The parameters `label` and `input` do not exist anymore.

Reply:
Yes, you are right, now we are expecting an array of inputs or labels, so we need to check whether the array is empty or not. Updated https://github.com/shopware/shopware/pull/4582/commits/a4fbbdd589b50dff60f3120e50674048727eab8c

Reply:
Thanks for the quick update. 👍 Since we are using `DomAccess.querySelectorAll` this will return `false` instead of an empty NodeList. If no elements are found couldn't this lead to a `length of undefined` error because `labels` would be `bool false`?

I guess it can already work by keeping the negated check and to change it `labels` and `inputs` instead.

```js
(!label || !input)
(!labels || !inputs) // <-- already containing "false"?
```

Reply:
this screams for a test :grin:

Reply:
Fixed.